draft-ietf-quic-transport.md

```diff
@@ -1370,8 +1369,8 @@ abstraction.  Streams can be created either by the client or the server, can
 concurrently send data interleaved with other streams, and can be cancelled.
 QUIC's stream lifetime is modeled closely after HTTP/2's {{?RFC7540}}.  Streams
 are independent of each other in delivery order.  That is, data that is received
-on a stream is delivered in order within that stream, but there is no particular
-delivery order across streams.  Transmit ordering among streams is left to the
+on a stream is ordered within that stream via the frame offset, but there is no particular
+order across streams.  Transmit ordering among streams is left to the
 implementation.  QUIC streams are considered lightweight in that the creation
```

Comment:
Not your change, but it seems a bit verbose to talk about how lightweight streams are 3 times in one paragraph.  Maybe we can remove the last sentence("Implementations are therefore advised...")?